src/pages/Admin/Admin.tsx

```diff
@@ -52,6 +52,7 @@ import { useSelector } from "hooks/useSelector";
 import { useShowHide } from "hooks/useShowHide";
 import { useUser } from "hooks/useUser";
 import { useVenueId } from "hooks/useVenueId";
+import { WorldUsersProvider } from "hooks/users/useWorldUsers";
```

Comment:
Can you please add an export of `WorldUsersProvider` to `src/hooks/users/index.ts` then import it like this:

```suggestion
import { WorldUsersProvider } from "hooks/users";
```

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1757/commits/26699fa3ad119469e8647bf1b17ea10b4d76eade

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1757/commits/26699fa3ad119469e8647bf1b17ea10b4d76eade